app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -38,6 +38,17 @@ examples:
       suggestion_link_text: "You can now apply for a permit online."
       suggestion_link_url: "/permit"
 
+  open_suggestion_link_in_new_tab:
+    description: |
+      For example if you wanted to drive users to another online task, you don't want to completely hijack
```

Comment:
Nitpick: I don't think you need `For example` at the start of this sentence, it's perfectly clear without it.

Reply:
Not a nitpick - thanks for the spot!